### DISCUSSION TITLE
Add composer.lock to repository

### DISCUSSION THREAD

**Discussion by cavestri** [openclassify/openclassify#1008] [General]
Hi, I am trying to install the script on Windows and I am constantly getting an error:

Fatal error: Allowed memory size of 4294967296 bytes exhausted (tried to allocate 4096 bytes)

I have tried change the memory_limit  to different values including -1 but still get a similar error.

Sometimes the install hangs with:

Something's changed, looking at all rules again (pass #110)

I guess that if you add the composer.lock file to the repo I could use it to install the script skipping all the composer dependencies checks because it will already have the exact libraries that needs to install.

Makes sense?

Thanks!

**Comment by fatihalp** [comment]
There is no such issue in composer 2.0 

Can you check again? @cavestri
